### PR TITLE
Update the code review process documentation to cover Reviewable.

### PR DESCRIPTION
**Pull request by david-german-tri**
Before asking Russ to review this, I'm going to leave it open a couple of days for comments.

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/2119)

### CODE REVIEW

**Comment by sherm1** [comment]
Why can't I review this with Reviewable?

File: drake/doc/reviewable.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+rebase freely without corrupting the review history.
+
+Life of a Reviewable Comment
+============================
+
+All threads in Reviewable must be resolved before you can merge your PR. The
+semantics of discussion resolution are a little tricky:
+
+When a reviewer creates a new comment, the reviewer's disposition toward the
+thread may be "satisfied", discussing", or "blocking".
```
Comment (sherm1) [inline drake/doc/reviewable.rst]: missing quote

**Comment by sherm1** [comment]
:+1:

File: drake/doc/reviewable.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+* Use **satisfied** to indicate that no action is required of the author.
```
Comment (liangfok) [inline drake/doc/reviewable.rst]: The previous sentence uses the phrase "When a reviewer" whereas these bullets is written as if they were talking to the reviewer. Consider adding "As a reviewer, you should..." before this bullet.

File: drake/doc/reviewable.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+* Use **discussing** to indicate that a response is required, but the author
```
Comment (liangfok) [inline drake/doc/reviewable.rst]: Same for this bullet: "As a reviewer, ..."

File: drake/doc/reviewable.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  may close the thread at their discretion without further input from you.
+  If the author has follow-up questions or wants you to take another look,
+  you are responsible for iterating on it. This is the default.
+* Use **blocking** to indicate that the author must take action on the
```
Comment (liangfok) [inline drake/doc/reviewable.rst]: Maybe just add the words "As a reviewer, you should:" before this entire bulleted list.

File: drake/doc/reviewable.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  comment. You must then review the fix and iterate on it with the author
+  until you are willing to update your status to **satisfied**.
+
+The author may respond to each comment with the same set of dispositions.
+
+* Use **satisfied** to indicate that you believe the discussion is over.
```
Comment (liangfok) [inline drake/doc/reviewable.rst]: Before this bulleted list, add "As an author of a PR, you should mark a comment as:"

File: drake/doc/reviewable.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+* Use **blocking** to indicate that you have not yet resolved the problem.
+  This serves as a safeguard against accidental merge.
+
+Before commenting on a line of code, reviewers should check to see if there
+is already a resolved discussion addressing the same topic. Resolved
+discussions are indicated by a small white check-mark in a grey circle to
+the left of the line of code.
+
+Reviewers should click the eye-shaped buttons to indicate that they have
+reviewed a file.  Reviewable will then remember the revision at which it
```
Comment (liangfok) [inline drake/doc/reviewable.rst]: "revision" --> "revisions"

**Comment by david-german-tri** [comment]
You can!  It's here, same as always: https://reviewable.io/reviews/RobotLocomotion/drake/2119#-

---

Review status: 0 of 2 files reviewed at latest revision, 7 unresolved discussions.

---

*[drake/doc/reviewable.rst, line 37 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFfQw8y4vU0GDRjY4vQ-r1-37:-KFim0-3BGvjSXJq22lL:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/3e1d3a6392441e4161ef08a3e891505e8abf2fbc/drake/doc/reviewable.rst#L37)):*
Done.

---

*[drake/doc/reviewable.rst, line 43 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFfQw8y4vU0GDRjY4vQ-r1-43:-KFimgmXPL_htnFpxj82:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/3e1d3a6392441e4161ef08a3e891505e8abf2fbc/drake/doc/reviewable.rst#L43)):*
Done.

---

*[drake/doc/reviewable.rst, line 56 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFfQw8y4vU0GDRjY4vQ-r1-56:-KFimqZVfAgYu6gjdVy7:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/3e1d3a6392441e4161ef08a3e891505e8abf2fbc/drake/doc/reviewable.rst#L56)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-:-KFilaJfpYtbG5Iu_e_u:-760994270)*

**Comment by david-german-tri** [comment]
Review status: 0 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[drake/doc/reviewable.rst, line 30 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFfQw8y4vU0GDRjY4vQ-r1-30:-KFinSHQDWZQLh8eXVhc:-2051389027) ([raw file](https://github.com/robotlocomotion/drake/blob/3e1d3a6392441e4161ef08a3e891505e8abf2fbc/drake/doc/reviewable.rst#L30)):*
Where?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119)*

**Comment by jwnimmer-tri** [comment]
*[drake/doc/reviewable.rst, line 30 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFfQw8y4vU0GDRjY4vQ-r1-30:-KFio1n_NnQs9yeFekSc:725810770) ([raw file](https://github.com/robotlocomotion/drake/blob/3e1d3a6392441e4161ef08a3e891505e8abf2fbc/drake/doc/reviewable.rst#L30)):*
Lead leading quote on `discussing"`.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119)*

**Comment by liangfok** [comment]
FTR, when this PR was first issues, the purple Reviewable button was not automatically added to the end of the PR's description. That is why I did not use Reviewable to leave my comments last night.

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-:-KFit7co8Ari6jzD_PQW:-411423320)*

**Comment by david-german-tri** [comment]
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[drake/doc/reviewable.rst, line 30 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFfQw8y4vU0GDRjY4vQ-r1-30:-KFj2xjmcDZbOBQHugzG:-570620961) ([raw file](https://github.com/robotlocomotion/drake/blob/3e1d3a6392441e4161ef08a3e891505e8abf2fbc/drake/doc/reviewable.rst#L30)):*
Done, thanks!  Amazing how stuff like that is impossible to see.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119)*

**Comment by david-german-tri** [comment]
Yeah, I think I deleted it by accident in the GitHub UI.  Sorry about that!

---

Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-:-KFj3B2iKsBzsOhUoPWA:1154662539)*

**Comment by liangfok** [comment]
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---

*[drake/doc/reviewable.rst, line 34 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFj3TGlHQ9_xqm5X3Ks:-KFj3TGlHQ9_xqm5X3Kt:-619111800) ([raw file](https://github.com/robotlocomotion/drake/blob/0a9a25540a7e0f6277cfdd000fa171a9686bf791/drake/doc/reviewable.rst#L34)):*
Since "author" is singular, should "their" be "his or her" instead?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119)*

**Comment by david-german-tri** [comment]
Review status: 1 of 2 files reviewed at latest revision, 4 unresolved discussions.

---

*[drake/doc/reviewable.rst, line 34 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFj3TGlHQ9_xqm5X3Ks:-KFj4aeG2sbewvrrLpEn:609073777) ([raw file](https://github.com/robotlocomotion/drake/blob/0a9a25540a7e0f6277cfdd000fa171a9686bf791/drake/doc/reviewable.rst#L34)):*
Rephrased to avoid.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119)*

**Comment by david-german-tri** [comment]
Review status: 1 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[drake/doc/reviewable.rst, line 32 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFfQw8y4vU0GDRjY4vQ-r1-32:-KFj4gsvGlbSAW5DYDmm:-687354696) ([raw file](https://github.com/robotlocomotion/drake/blob/3e1d3a6392441e4161ef08a3e891505e8abf2fbc/drake/doc/reviewable.rst#L32)):*
Now moot.

---

*[drake/doc/reviewable.rst, line 33 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-KFfQw8y4vU0GDRjY4vQ-r1-33:-KFj4jmM26qs1o3IVzMI:-687354696) ([raw file](https://github.com/robotlocomotion/drake/blob/3e1d3a6392441e4161ef08a3e891505e8abf2fbc/drake/doc/reviewable.rst#L33)):*
Now moot.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119)*

**Comment by sherm1** [comment]
:lgtm:

---

Review status: 1 of 2 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-:-KFj6E5Ny2DwHE6sR7pI:1417088721)*

**Comment by sherm1** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119)*

**Comment by liangfok** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-:-KFj7MO7manaYP_cUH0s:1417088721)*

**Comment by liangfok** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119)*

**Comment by david-german-tri** [comment]
+@russtedrake 

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2119#-:-KFkUmp5GHvrzrxgv0mQ:-984191903)*

**Pull request merged**